wandb/sdk/launch/docker.py

```diff
@@ -157,15 +157,14 @@ def build_docker_image_if_needed(
     container_env: List[str],
     runner_type: str,
     image_uri: str,
-    command_args: List[str],
+    command_args: Optional[List[str]],
```

Comment:
I don't think we want this to be optional, because we always want to save the command we ran in the metadata file